### DISCUSSION TITLE
How does one configure Azure Open AI for assistant?

### DISCUSSION THREAD

**Discussion by wzhliang** [zed-industries/zed#16444] [Feature Requests]
Tried to follow screenshot in PR but didn't work

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
